roles/explorer.nix

```diff
@@ -11,6 +11,8 @@ in {
     (sources.cardano-node + "/nix/nixos")
     (sources.cardano-explorer + "/nix/nixos")
   ];
+  services.graphql-engine.enable = true;
+  services.cardano-graphql.enable = true;
```

Comment:
The manual steps mentioned in the PR description should be handled automatically in these services.

Reply:
Confirmed that they are. Removed that caveat from the PR.